html/pages/oxidized.inc.php

```diff
@@ -30,6 +30,8 @@
                             <thead>
                             <tr>
                                 <th data-column-id="hostname" data-order="desc">Hostname</th>
+                                <?php echo '<th data-column-id="sysname" data-visible="' . (!Config::get('force_ip_to_sysname')  ? 'true' : 'false') . '">SysName</th>';
```

Comment:
you could move most of the html outside the php tags, just the true/false ternary needs to be inside <?php